flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
so aarch64 are denied this? I thought Electron was cross platform.

Disappointing

Reply:
Currently I do not have a machine to actually test this app for aarch64.
Only allow build for aarch64 is still ok ?

Reply:
There is a binary dependency (librush) currently only tested under x86_64, so it may not work under aarch64.